packages/rest/src/utils/agent.ts

```diff
@@ -0,0 +1,56 @@
+import type { TestLogger } from './logger'
+import type { InitConfig, AutoAcceptCredential } from '@aries-framework/core'
+
+import { Agent, HttpOutboundTransport } from '@aries-framework/core'
+import { agentDependencies, HttpInboundTransport } from '@aries-framework/node'
+
+import { BCOVRIN_TEST_GENESIS } from './util'
+
+export async function setupAgent({
```

Comment:
Is this method still used? I think it should be moved to tests or samples directory